test/extended/authorization/authorization.go

```diff
@@ -1241,3 +1242,93 @@ var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] authorization", f
 		})
 	})
 })
+var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] ImageRegistry access", func() {
+    defer g.GinkgoRecover()
+    oc := exutil.NewCLI("bootstrap-policy")
+
+    g.Context("", func() {
+        g.Describe("PublicImageAccessWithBasicAuthShouldSucceed", func() {
+            g.It("should succeed [apigroup:image.openshift.io]", func() {
+
+                g.By("Enable default route for the image registry")
+                err := oc.AsAdmin().WithoutNamespace().Run("patch").Args(
+                    "configs.imageregistry.operator.openshift.io/cluster",
+                    "--type=merge",
+                    "-patch", `{"spec": {"defaultRoute": true}}`,
+                ).Execute()
+                o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
Consider using `waitRouteReady` here to make sure you only continue your tests once the route is fully provisioned. Sometimes provisioning a route can take a second or two, and you might end up with some flake runs of this if you don't wait.

The default route's name is `default-route`.